packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/add-ip-instance/controller.js

```diff
@@ -0,0 +1,76 @@
+export default class OctaviaLoadBalancerPoolsDetailMembersAddIpInstanceCtrl {
+  /* @ngInject */
+
+  constructor(Alerter, OctaviaLoadBalancerMembersService, $translate) {
+    this.Alerter = Alerter;
+    this.OctaviaLoadBalancerMembersService = OctaviaLoadBalancerMembersService;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.model = {
+      checked: [],
+    };
+    this.instances = [];
+    this.displayedInstances = [];
+    this.resetCurrentStep();
+    this.loadIpInstances();
+  }
+
+  loadIpInstances() {
+    this.isLoading = true;
+    this.OctaviaLoadBalancerMembersService.getInstances(this.projectId)
+      .then((instances) => {
+        // For each instance, we create a new instance per ip address
+        // We create a new instance only for IPV4 ip address
+        angular.forEach(instances, (instance, index) => {
+          this.computeIpV4Instance(instance);
+          this.model.checked[index] = false;
```

Comment:
This should probably be on computeIpV4Instance function forEach to be in sync with instances array. To be fix on the next PR :)